cli/test/test_run_parser.py

```diff
@@ -48,6 +49,53 @@ def test_run_parser_local(test_input_dir) -> None:
         for output_file in Path(output_dir).glob("*.json"):
             assert ParserOutput.parse_file(output_file)
 
+            if "html" in str(output_file):
+                html_data = ParserOutput.parse_file(output_file).html_data
+                assert html_data.text_blocks != [] or html_data.text_blocks is not None
+
+            if "pdf" in str(output_file):
+                pdf_data = ParserOutput.parse_file(output_file).pdf_data
+                assert pdf_data.text_blocks != [] or pdf_data.text_blocks is not None
+                assert pdf_data.md5sum != ""
+                assert (
+                    pdf_data.page_metadata != [] or pdf_data.page_metadata is not None
+                )
```

Comment:
`not in {[], None}` or just `assert pdf_data.text_blocks` (and the same for `page_metadata`)?